Update fileio.rst (#753)
ansys/pyprimemesh: doc/source/user_guide/fileio.rst

Four CAD import routes are available in PyPrimeMesh:

 * ``Program controlled``: chooses the CAD reader route based on the extension of the provided CAD file, as follows;
   native for FMD, SAT, SAB, X_T, X_B, JT, PLMXML, and STL;
   SpaceClaim for SCDOC and SCDOCX;
   Discovery for DSCO;
   Workbench for all other extensions.

>  * ``Native``: natively supported file format extensions are FMD, ACIS (SAT and SAB),

Reviewer: ```suggestion
 * ``Native``: Natively supported file format extensions are FMD, ACIS (SAT and SAB),
```

   Parasolid (X_T and X_B), JTOpen (JT and PLMXML), and STL.

>  * ``SpaceClaim``: uses SpaceClaim to import supported CAD files from the SpaceClaim reader. Only the

Reviewer: ```suggestion
 * ``SpaceClaim``: Uses SpaceClaim to import supported CAD files from the SpaceClaim reader. Only the
```

   Windows platform supports importing files using the SpaceClaim reader.

>  * ``Discovery``: uses Discovery to import supported CAD files from the Discovery reader. Only the
>    Windows platform supports importing files using the Discovery reader.
> 
>  * ``Workbench``: uses Workbench to import supported CAD files from the Workbench reader.

Reviewer: ```suggestion
 * ``Discovery``: Uses Discovery to import supported CAD files from the Discovery reader. Only the
   Windows platform supports importing files using the Discovery reader.

 * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
```

> To view the CAD supported for the Workbench route on different platforms, see

Reviewer: ```suggestion
To view the CAD files supported for the Workbench route on different platforms, see
```

`CAD Support <https://www.ansys.com/it-solutions/platform-support>`_ on the Ansys website. 

.. note::
    * Program controlled and SpaceClaim CAD reader routes import JT files containing only faceted data. Workbench and
      Native CAD reader routes do not support importing JT files containing only faceted data.

    * When deploying scripts using SpaceClaim, Discovery or Workbench CAD reader routes, ensure that the CAD configuration
      and in-app defaults are consistent in the deployed environment.

    * You must install and configure Workbench CAD readers or plug-ins (Ansys Geometry Interfaces)
      while installing Ansys Workbench.

    * In order to preserve shared topology the Workbench CAD reader route must be used.

    * Patterned name selections are not supported for the SpaceClaim or Discovery CAD reader routes.

>       To import patterned named selections the Workbench CAD reader route can be used.

Reviewer: ```suggestion
      To import patterned named selections, the Workbench CAD reader route can be used.
```

 * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.

>  * ``Discovery``: Uses Discovery to import supported ``*.dsco`` and  ``*.scdocx`` files.

Reviewer: ```suggestion
 * ``Discovery``: Uses Discovery to import supported DSCO and SCDOCX files.
```
Style guide says to use all capital letters for file extensions when used standalone.

To view the CAD supported for the Workbench route on different platforms, see
`CAD Support <https://www.ansys.com/it-solutions/platform-support>`_ on the Ansys website. 

.. note::

> *   Program Controlled and SpaceClaim route imports JT files containing only faceted data. Workbench and

Reviewer: Having trouble understanding the initial capital letters in the first few words here and in the next few lines. Only capitalize proper nouns. Any UI elements in a product should be in bold font.

Reviewer 2: @PipKat resolving as outdated. LGTM

    Native does not support importing JT files containing only faceted data.

> *   Transmogrifier supports files from Workbench(pmdb,agdb), JT(faceted data from ``(*.jt/*.plmxml)``,

Reviewer: ```suggestion
*   Transmogrifier supports Workbench PMDB and AGDB files, JT faceted data JT and PLMXML files, 
```
I'm not sure about that second part, but that partly demonstrates how the current format isn't all that clear.

	XC ``(.fmdb, .sab/.sat, .stl, .msh/.tgf, .x_t/.x_b)``.

> *  SpaceCliam supports files with ``*.scdoc``, ``*.scdocx`` extension.
> 
> *  Discovery supports files with ``*.dsco``, ``*.scdocx`` extension.

Reviewer: ```suggestion
*  SpaceCliam supports files with SCDOC and SCDOCX extensions.

*  Discovery supports files with DSCO and SCDOCX extensions.
```
